lib/graphiql/rails.rb

```diff
@@ -18,14 +18,44 @@
 
 module GraphiQL
   module Rails
+    class Config
+      attr_accessor :query_params, :initial_query, :headers
+
+      def initialize(query_params:, initial_query:, headers:)
+        @query_params = query_params
+        @initial_query = initial_query
+        @headers = headers
+      end
+
+      def csrf=(_)
+        raise %|
+
+csrf option has been removed, set the header explicitly:
+
+  GraphiQL::Rails.config.headers['X-CSRF-Token'] = ->(context) do
+    context.form_authenticity_token
+  end
```

Comment:
Is it possible to re-implement this feature by assigning `X-CSRF-Token` during `resolve_headers` if this config is true? 

If it's possible to re-implement, would you mind changing the default to `true` while you're at it? I think I chose the wrong default the first time around!